gr-pdu/examples/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Jacob Gilbert
+#
+# This file is part of GNU Radio
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+install(
+  FILES
+  DESTINATION ${GR_PKG_DATA_DIR}/examples/pdu
```

Comment:
It would be nice to have some example .grc files for gr-pdu in here.  Shouldn't hold up the merge but would be a good part of the overall package.

Reply:
I agree. I'll build some with the next addition of blocks.